<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-F8PE7CG" failures="0" tests="2" name="Default test" time="0.048" errors="0" timestamp="2020-05-23T22:46:51 EDT">
  <testcase classname="TestNG_hsn.DataProvider__Recap3__SoftAssert" name="@BeforeMethod login" time="0.002">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestNG_hsn.DataProvider__Recap3__SoftAssert.login(DataProvider__Recap3__SoftAssert.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeMethod login -->
  <testcase classname="TestNG_hsn.DataProvider__Recap3__SoftAssert" name="@AfterMethod logaout" time="-1.590288411135E9">
    <skipped/>
  </testcase> <!-- @AfterMethod logaout -->
  <testcase classname="TestNG_hsn.DataProvider__Recap3__SoftAssert" name="@BeforeMethod login" time="-1.590288411136E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod login -->
  <testcase classname="TestNG_hsn.DataProvider__Recap3__SoftAssert" name="@AfterMethod logaout" time="-1.590288411139E9">
    <skipped/>
  </testcase> <!-- @AfterMethod logaout -->
  <testcase classname="TestNG_hsn.DataProvider__Recap3__SoftAssert" name="@AfterClass closeBrowser" time="-1.590288411141E9">
    <skipped/>
  </testcase> <!-- @AfterClass closeBrowser -->
  <testcase classname="TestNG_hsn.DataProvider__Recap3__SoftAssert" name="addEmployee" time="0.0">
    <skipped/>
  </testcase> <!-- addEmployee -->
  <testcase classname="TestNG_hsn.DataProvider__Recap3__SoftAssert" name="addEmployee" time="0.0">
    <skipped/>
  </testcase> <!-- addEmployee -->
</testsuite> <!-- Default test -->
